UserManagement/accounts/domain/models.py

```diff
@@ -37,7 +50,9 @@ class Meta:
         app_label = 'accounts'
 
     def __str__(self):
-        user_full_name = self.user.full_name if self.user else "Unknown user"
-        return f"{user_full_name}'s Rating: {self.rate}"
+        user_full_name = self.user_id.full_name if self.user_id else "Unknown user"
```

Comment:
Why does the "user_id" have the "full_name" ? (Naming error ?)

Reply:
Vitalik tell that just user is bad naming 😵